docs/CONTRIBUTING.rst

```diff
@@ -1,11 +0,0 @@
-=========================
-Contributing to OpenHatch
-=========================
-
-Before opening a pull request or submitting a patch, please `read the guidelines for contributing`_.
-
-If you submit more than one pull request, you'll find that it's helpful to use a topic branch to avoid duplicating commits.
-
-.. _read the guidelines for contributing: http://openhatch.readthedocs.org/en/latest/getting_started/getting_started.html
-
-.. Further changes to this file should be made using the limited subset of rST syntax that GitHub supports.
```

Comment:
+1 on this deletion -- I think the one at the root makes sense, but the one in docs/ doesn't.